scenarios/20_Okladia.cfg

```diff
@@ -93,7 +93,7 @@
         {ELSE_IF ano_strateg_pos equals 2} #DEORIEN
         {TELEPORT (Gawen Hagarthen) 15 16}
         {IF_HAVE_UNIT (Deorien)}
-        # (nothing to do... maybe add a message?)
+        # (nothing to do... maybe add a message? Except no, this is a prestart event, so it wouldn't show...)
```

Comment:
I guess you could just delete this comment then?

Reply:
well I can trim it down at least

Reply:
shortened in [efba779](https://github.com/nemaara/A_New_Order/pull/43/commits/efba77972af8c9bcc1fc309bcc70c2a6748d2016)